### DISCUSSION TITLE
Error : a target file must be specified

### DISCUSSION THREAD

**Discussion by Ruegnos** [prominic/Moonshine-IDE#1252] [Q&A] [answered]
Hi everyone,

I want to build and debug my project with a mxml file but moonshine give me this error when I try to.
I work on MacOs and I don't see the option for add a target file.
thanks for your help,
![image](https://github.com/Moonshine-IDE/Moonshine-IDE/assets/152271849/ac508d43-f785-4c7c-a206-d1843dcfd64a)

**Comment by Ruegnos** [comment, marked as answer]
Finally you  just have to restart your IDE when you edit the settings

**Comment by rat-moonshine** [comment]
Hello @Ruegnos , it's good to hear that you somehow able to resolve the problem you were facing. 

However, it's not necessary to restart IDE after you edit the settings - a Save suppose to do necessary modification once you edit settings. Can you please give us some input what have you edited in settings that didn't affect post Save?

**Reply by Ruegnos** [reply]
Hello,
 I was saving the settings every time I made a change, but the error kept coming back. I didn’t change anything in my settings, but I just restarted the IDE, and it worked!
